src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java

```diff
@@ -129,14 +130,16 @@ public void shootAtComputer(int row, int col) {
     }
 
     public void shootAtPlayer() {
-        int max = 10;
-        int min = 1;
-        Random random = new Random();
-        int randRow = random.nextInt(max - min + 1) + min;
-        int randCol = random.nextInt(max - min + 1) + min;
-
-        Coordinate coor = new Coordinate(randRow,randCol);
-        playerShot(coor);
+        //this is easy mode, goes from one to 10 then moves columns
+            easyRow++;
+            if(easyRow%10==0){
+                easyRow=1;
+                easyCol++;
+            }
+           Coordinate coor = new Coordinate(easyRow, easyCol);
```

Comment:
I see that you're hard coding the fire location so the computer fires every spot and has to win at some point. However, this has to be implemented with object oriented design and the function needs to be used only when the user selects easy computer difficulty.